pkg/entity/fixture.go

```diff
@@ -80,3 +82,47 @@ func GenFixtureSegment() Segment {
 	s.PrepareEvaluation()
 	return s
 }
+
+func GenFixtureSegmentWithAdditionalProperty(propertyName string, entityID string) Segment {
```

Comment:
this helper function is too specific that the input parameter doesn't match the name of the function.

how about we reuse `GenFixtureSegment()` in the test, and then just manually add a new constraint the following in the tests? This makes the test looks closer to what exactly it's testing, what do you think?


```
s := GenFixtureSegment()
s.Constraints = append(s.Constraints, 			{
				Model:     gorm.Model{ID: 501},
				SegmentID: 200,
				Property:  propertyName,
				Operator:  models.ConstraintOperatorEQ,
				Value:     fmt.Sprintf(`"%s"`, entityID),
			})

s.PrepareEvaluation()
```

Reply:
Totally agree, implemented this suggestion and got rid of the additional fixture.